test/install.test.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+const assert = require("assert");
+const { execSync } = require("child_process");
+const fs = require("fs");
+const os = require("os");
+process.env.DEBUG = "aio-asset-compute*";
+
+function shell(cmd, dir) {
+    cmd = cmd
+        .split("\n")
+        .map(line => line.trim())
+        .filter(line => line.length > 0)
+        .filter(line => !line.startsWith("#"))
+        .join(os.platform() === "win32" ? " & " : "; ");
+    execSync(cmd, {cwd: dir, stdio: 'inherit'});
+}
+
+it("should install (local) version of aio-cli and run developer experience", async function() {
```

Comment:
can't this test just replace this one: https://github.com/adobe/asset-compute-integration-tests/blob/e8c5321054ea742ddea38f3647422f59cec4abe2/test/index.test.js#L64?

Reply:
It could, however I was trying to do this differently by installing the cli locally.  If you look at the output of `npx aio info` you can see what that the versions in use can differ from the global install. 
So this is not **_exactly_** the same, but very close.